### DISCUSSION TITLE
Airflow randomly crashes after upgrade to 2.9.3: cannot execute INSERT in a read-only transaction

### DISCUSSION THREAD

**Discussion by cesar-vermeulen** [apache/airflow#40852] [Q&A]
Hey everybody,

Currently opening this as a discussion as I'm not sure this is linked to our Postgres backend our the airflow deployment - if it's a better option to convert this to an issue, I'm happy to do so!

Edit: solved - not linked to Airflow

**Comment by niftash** [comment]
I'm facing similar issue, how did you solve it ?

**Comment by cesar-vermeulen** [comment]
Hello @niftash - our backend database's storage got to 98% and went into read-only mode. Cleaning airflow metadata database was the solution :)
